src/pkg/egress/syslog/syslog_connector.go

```diff
@@ -131,11 +131,16 @@ func (w *SyslogConnector) Connect(ctx context.Context, b Binding) (egress.Writer
 		w.droppedMetric.Add(float64(missed))
 		drainDroppedMetric.Add(float64(missed))
 
-		w.emitErrorLog(b.AppId, fmt.Sprintf("%d messages lost in user provided syslog drain", missed))
-
+		if b.AppId != "" {
+			w.emitErrorLog(b.AppId, fmt.Sprintf("%d messages lost in user provided syslog drain", missed))
+		}
+		errorAppOrAggregate := fmt.Sprintf("for %s's app drain", b.AppId)
+		if b.AppId == "" {
+			errorAppOrAggregate = "for aggregate drain"
+		}
 		log.Printf(
-			"Dropped %d %s logs for url %s in app %s",
-			missed, urlBinding.Scheme(), anonymousUrl.String(), b.AppId,
+			"Dropped %d %s logs %s with url %s",
```

Comment:
Also, I'm a bit concerned that there were no tests to update for this. Maybe we should add some..

Reply:
Possibly. This is for standard out, so I think it's less likely/suggested that people have automated this part to me. There arn't tests for this because standard out.